docs/source/development/traffic_ops_api/v12/deliveryservice.rst

```diff
@@ -2648,7 +2648,266 @@ SSL Keys
     }
 
 |
+**PUT /api/1.2/deliveryservices/{:id}/safe**
 
+  Allows a portal user to edit limited fields of an assigned delivery service.
+
+  Authentication Required: Yes
+
+  Role(s) Required:  portal or above
```

Comment:
can you update this? what's the rule? ops or above or deliveryservice is assigned to the user?